packages/account/src/providers/operations.graphql

```diff
@@ -61,10 +61,64 @@ fragment transactionStatusFragment on TransactionStatus {
   }
 }
 
+fragment malleableFieldsFragment on Transaction {
+  receiptsRoot
+  inputs {
+    type: __typename
+    ... on InputCoin {
+      txPointer
+    }
+    ... on InputContract {
+      txPointer
+    }
+  }
+  outputs {
+    type: __typename
+    ... on CoinOutput {
+      to
+      amount
+      assetId
+    }
+    ... on ContractOutput {
+      inputIndex
+      balanceRoot
+      stateRoot
+    }
+    ... on ChangeOutput {
+      to
+      amount
+      assetId
+    }
+    ... on VariableOutput {
+      to
+      amount
+      assetId
+    }
+    ... on ContractCreated {
+      contract
+      stateRoot
+    }
+  }
+}
```

Comment:
@nedsalk Why not just get the `rawPayload` property for both `SuccessStatus` and `FailureStatus`?

Wouldn't that avoid assembling a processed TX manually later on?

Reply:
The `rawPayload` property only exists on `Transaction`, there is none on the statuses. If you mean to fetch that property, then there wouldn't be any bandwidth benefit as I'd be fetching the whole transaction.

Reply:
@nedsalk Maybe I'm missing something, but wasn't the transaction added to both the success and failure statuses?

https://github.com/FuelLabs/fuels-ts/blob/5727b17913ad7a9f5342d0d7ad057735c1d442ed/packages/account/src/providers/fuel-core-schema.graphql#L1247-L1257

https://github.com/FuelLabs/fuels-ts/blob/5727b17913ad7a9f5342d0d7ad057735c1d442ed/packages/account/src/providers/fuel-core-schema.graphql#L1300

Reply:
My idea was to have the `rawPayload` property available within a `submitAndAwait`/`statusUpdate` response to avoid manually assembling the TX

Reply:
Yes, but we don't want to fetch the whole transaction via `rawPayload` because we already have most of it locally via `TransactionRequest.toTransaction`. We just want to fetch the _malleable_ fields who's values are set by the node.